tests/data/test_extension/sources.txt

```diff
@@ -0,0 +1,3 @@
+
+cruise-ship-horn-sound.mp3: https://www.freesoundslibrary.com/cruise-ship-horn-sound/
+
```

Comment:
This file is licensed under CC-BY-4.0. That's fine, but this changeset fails to meet the basic [terms](https://creativecommons.org/licenses/by/4.0/) of that license.

Reply:
OK ... created my own mp3 and removed the cruise ship.